libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -147,6 +147,76 @@ static IotMqttError_t _subscriptionCommon( IotMqttOperationType_t operation,
                                            const IotMqttCallbackInfo_t * pCallbackInfo,
                                            IotMqttOperation_t * const pOperationReference );
 
+typedef IotMqttError_t ( * _pingreqSerializer_t )( uint8_t **, size_t * );
```

Comment:
Probably better to move these typedefs to iot_mqtt_types.h and then the definition of ```IotMqttSerializer_t``` can also use those.

Reply:
Good suggestion, will change and upload new version.